### ISSUE TITLE
TypeError converting to SARIF

### ISSUE THREAD

**Issue by ksolie** [veracode/veracode-pipeline-scan-results-to-sarif#10]
```
/home/runner/work/_actions/Veracode/veracode-pipeline-scan-results-to-sarif/master/dist/index.js:640
    core_1.default.setFailed(error.message);
                   ^

TypeError: Cannot read property 'setFailed' of undefined
    at Object.672 (/home/runner/work/_actions/Veracode/veracode-pipeline-scan-results-to-sarif/master/dist/index.js:640:20)
    at __nccwpck_require__ (/home/runner/work/_actions/Veracode/veracode-pipeline-scan-results-to-sarif/master/dist/index.js:800:43)
    at /home/runner/work/_actions/Veracode/veracode-pipeline-scan-results-to-sarif/master/dist/index.js:817:18
    at Object.<anonymous> (/home/runner/work/_actions/Veracode/veracode-pipeline-scan-results-to-sarif/master/dist/index.js:818:12)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
    at internal/main/run_main_module.js:17:11
 ```

**Comment by githubrlloyd** [comment]
thanks - looking at this, tried to improve our SARIF conversion, will close this as soon as its resolved

**Comment by githubrlloyd** [comment]
This should be resolved with the v0.1.6 release that is available. You can reference the action @master or by version to minimize any breaking changes impacting your workflow.
